CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 6.17.9
+
+- fix(gatsby): Add missing React peer dependency ([#4576](https://github.com/getsentry/sentry-javascript/pull/4576))
```

Comment:
@AbhiPrasad do we want to thank external contributor for this PR https://github.com/getsentry/sentry-javascript/pull/4576

Reply:
Yeah we should make the edit